websites/R/Roblox/presence.ts

```diff
@@ -4,10 +4,12 @@ const presence = new Presence({
   browsingTimestamp = Math.floor(Date.now() / 1000);
 let profileName,
   profileTabs,
+  profileAvatar,
   messageTab,
   friendsTab,
   inventoryTab,
   groupName,
+  groupImage,
```

Comment:
Why not `const`?

Reply:
For the same reason the other variables aren't? I only matched the code/format style the file was already in.